index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <header>
+            <div class="logo-placeholder">John Doe</div>
+            <button type="button"><img src="img/Union.png" alt="image of an icon, conformed by 3 lines"></button>
```

Comment:
- [ ] Kindly make sure that the hamburger button is on the right and has a nav tag as stated in the project requirements